packages/components/src/QualityBar/QualityRatioBar.component.js

```diff
@@ -37,14 +45,22 @@ export function QualityInvalidLine({ value, percentage, t }) {
 				percentage,
 				value: formatNumber(value),
 			})}
+			dataFeature={dataFeature}
+			onClick={onClick ? e => onClick(e, paramOnClick) : null}
+			ref={fowardRef}
```

Comment:
you can't juste use a `ref`props,: https://reactjs.org/docs/refs-and-the-dom.html#exposing-dom-refs-to-parent-components

If you try in your storybook, you'll see your ref is empty

![image](https://user-images.githubusercontent.com/58977230/103560656-6592ca80-4eb8-11eb-9eb6-ef98a7f2b126.png)

Reply:
when thinking about it, we don't even need a ref, we can use the event.target